mesh/mesh.cpp

```diff
@@ -11188,14 +11191,17 @@ void Mesh::Printer(std::ostream &os, std::string section_delimiter) const
    }
 }
 
-void Mesh::PrintTopo(std::ostream &os,const Array<int> &e_to_k) const
+void Mesh::PrintTopo(std::ostream &os, const Array<int> &e_to_k,
+                     std::string const& comments) const
```

Comment:
Re: using `const std::string` instead of `std::string const`:

There are more places, like this one, where `std::string const` is used -- I counted 15 places. 😁